packages/vscode-ext/tsconfig.json

```diff
@@ -2,12 +2,20 @@
   "compilerOptions": {
     "module": "commonjs",
     "target": "es2020",
-    "lib": ["es2020", "dom"],
+    "lib": [
+      "es2020",
+      "dom"
+    ],
     "outDir": "out",
     "rootDir": "src",
     "sourceMap": true
   },
-  "include": ["src"],
-  "exclude": ["node_modules", ".vscode-test"],
-  "references": [{ "path": "./client" }, { "path": "./server" }]
+  "include": [
+    "src/server/server.ts",
+    "src/client/extension.ts"
+  ],
```

Comment:
Maybe the entire [src/**/*](https://www.typescriptlang.org/tsconfig#include)? No need to hardcode names here.

Reply:
IMO having distinct entrypoints is nice — it will throw warnings on unused files iirc

Reply:
> it will throw warnings on unused files iirc

I checked and it doesn't.

One failure mode here is: during the refactoring in the future we could rename the server.ts to index.ts and create a new server.ts as a utility file, and a few other files. Then server.ts would get checked but other files wouldn't and we could easily miss this.

Reply:
done